rasa/nlu/featurizers/dense_featurizer/lm_featurizer.py

```diff
@@ -28,14 +31,22 @@
 
 logger = logging.getLogger(__name__)
 
-MAX_SEQUENCE_LENGTHS = {
-    "bert": 512,
-    "gpt": 512,
-    "gpt2": 512,
-    "xlnet": NO_LENGTH_RESTRICTION,
-    "distilbert": 512,
-    "roberta": 512,
+MODEL_WEIGHTS_DEFAULT = {
+    "bert": "rasa/LaBSE",
+    "gpt": "openai-gpt",
+    "gpt2": "gpt2",
+    "xlnet": "xlnet-base-cased",
+    "distilbert": "distilbert-base-uncased",
+    "roberta": "roberta-base",
 }
+CLS_TOKEN = "[CLS]"
+
+# The following are the prefix characters added to sub-tokens by the currently
+# used WordPiece, SentencePiece and Byte Pair tokenizers. See
+# https://huggingface.co/course/chapter6 for a description of each.
+# In case a new tokenizer uses different characters to split sub-tokens, this
```

Comment:
For clarification -- does this cover all the existing tokenizers? Or just the ones corresponding to the list we had above? I am not sure if our version of `tokenizers` (fairly old) has an additional tokenizer we need to worry about
<img width="726" alt="Screenshot 2022-01-19 at 10 38 13" src="https://user-images.githubusercontent.com/45405119/150104464-d6213f97-953f-49f9-854e-1fbb7dbbfa7d.png">

We should clarify "currently used" to make sure it's clear for users (currently used by us? currently existing?)

Is there a way to programmatically check the prefix characters? Maybe it is an attribute of the tokenizer, or something? I am not sure there is (seems like the `BaseTokenizer` class does not have an attribute), and SentencePiece calls these `replacement`, whereas WordPiece calls this `wordpieces_prefix`.

Reply:
Ah, I think maybe I was looking for `self.all_special_tokens` [here](https://huggingface.co/docs/transformers/v4.15.0/en/main_classes/tokenizer#transformers.PreTrainedTokenizer)?

Old docs won't load